config/locales/en/loans.en.yml

```diff
@@ -22,6 +22,7 @@ en:
     details: "Details"
     duplicate: "Duplicate"
     duplicated_message: "The loan was successfully duplicated."
+    confirm_duplication: "This will duplicate the %{object_type} and all associated timeline entries, logs, and questionnaire responses. Do you want to continue?"
```

Comment:
I can see why you'd do it this way but using parameters in this way doesn't always work in languages other than english due to genders, agreements, etc. e.g. in spanish, step might be masculine while group might be feminine, etc. Can you please add one full translation per object type, perhaps nested under `confirm_duplication`?